lib/Mojo/Log.pm

```diff
@@ -66,7 +67,20 @@ sub _default {
 sub _log {
   my ($self, $level) = (shift, pop);
   my @msgs = ref $_[0] eq 'CODE' ? $_[0]() : @_;
-  $msgs[0] = "$self->{context} $msgs[0]" if $self->{context};
+  if ($self->{context}) {
+    if (@msgs == 1 && blessed($msgs[0]) && $msgs[0]->isa('Mojo::Exception')) {
+      $msgs[0] = Mojo::Exception
+        ->new("$self->{context} ".$msgs[0]->message)
```

Comment:
There is no way this ever went through perltidy.

Reply:
Oops, missed that. Should be fixed now.